sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizePiiEntities.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics;
+
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Configuration;
+
+/**
+ * Sample demonstrates how to recognize the Personally Identifiable Information entities of document.
+ */
+public class RecognizePiiEntities {
+    /**
+     * Main method to invoke this demo about how to recognize the Personally Identifiable Information entities of
+     * document.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        TextAnalyticsClient client = new TextAnalyticsClientBuilder()
+            .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_API_KEY")))
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_ENDPOINT"))
+            .buildClient();
+
+        // The document that needs be analyzed.
+        String document = "My SSN is 555-55-5555";
```

Comment:
consider using `859-98-0987`. Same for other places

Reply:
why this SSN number, it seems like a more real SSN number that 55555555.

Reply:
is the one the service recommended us to use. You have a combination of that and 55555555 so I guess for consistency just choose one for all your test